game/src/layer/elevation.rs

```diff
@@ -46,31 +46,44 @@ impl Layer for Elevation {
     }
 }
 
-impl Elevation {
-    pub fn new(ctx: &mut EventCtx, app: &App) -> Elevation {
-        let mut colorer = ColorNetwork::new(app);
+impl SteepStreets {
+    pub fn new(ctx: &mut EventCtx, app: &App) -> SteepStreets {
+        let mut colorer = ColorDiscrete::new(
+            app,
+            vec![
+                // Colors and buckets from https://github.com/ITSLeeds/slopes
+                ("3-5% (almost flat)", Color::hex("#689A03")),
```

Comment:
There seems to be a mismatch in the buckets defined here and the buckets labeled below (staring on line 69).

Specifically, there is no 0-3 bucket here, and there is no 8-10 bucket there.

I think it ends up working out visually, but we should probably align the buckets to clarify the code.

Reply:
Good catch! I think it matches up now.